### DISCUSSION TITLE
1.0.0 installed from DMG on macOS X86_64

### DISCUSSION THREAD

**Discussion by neilyoung** [mucommander/mucommander#613] [Q&A] [answered]
...first system suggestion is to throw it away, since it is damaged...

**Comment by augustinluca** [comment]
same for me

**Reply by elias137** [reply]
same here

**Comment by ahadas** [comment, marked as answer]
yeah, that's a know issue - can you try `"sudo xattr -cr /Applications/muCommander.app"`?

**Reply by neilyoung** [reply]
Yes, that worked. Is there anywhere a change list? The readme mentions 0.9.7 or so

**Reply by augustinluca** [reply]
works for me too

**Reply by ahadas** [reply]
Thanks for the confirmation!

> Yes, that worked. Is there anywhere a change list? The readme mentions 0.9.7 or so

Should be included in the dmg but you can find it also [here](https://github.com/mucommander/mucommander/blob/release_1_0_0/package/readme.txt)

**Comment by ahadas** [comment]
As noted on #615, this issue seems to be different than what we had before - could be a result of the new packaging mechanism that v1.0.0 was created with (which is based on jpackage). It didn't happen on my machine though, which makes me think that in order to solve it we'll need to sign the application..

**Comment by thawn** [comment]
Just to improve findability, I am posting the error message text here: “muCommander.app” is damaged and can’t be opened.

**Reply by augustinluca** [reply]
Same error for me. The package is damaged

**Comment by micheleissa** [comment]
This is my work laptop and they don't give me sudo access. Is there any other workaround? I really want to try this app.

Thanks

**Reply by ahadas** [reply]
> This is my work laptop and they don't give me sudo access. Is there any other workaround? I really want to try this app.

I plan to sign the upcoming version but I don't have an estimation for when it will be released yet, hopefully in couple of weeks

**Comment by ahadas** [comment]
I didn't update the website yet but the artifacts, including the dmg, of v1.0.1 [have been uploaded](https://github.com/mucommander/mucommander/releases/tag/1.0.1-1) and the dmg is now signed and should not trigger this error anymore

**Comment by thawn** [comment]
I can confirm that v1.0.1 starts without error on my system. Thanks a lot for fixing this issue :) !

**Comment by superbonaci** [comment]
With macOS 13.1 new method is required. https://github.com/mucommander/mucommander/issues/850
